cmd/elasticindexer/config/prefs.toml

```diff
@@ -1,14 +1,18 @@
 [config]
     disabled-indices = []
     [config.web-socket]
-        server-url = "localhost:22111"
-        is-server = true
+        # URL for the WebSocket client/server connection
+        # This value represents the IP address and port number that the WebSocket client or server will use to establish a connection.
+        url = "localhost:22111"
+        # This flag will start the WebSocket connector as server or client( can be "client" or "server")
```

Comment:
This flag describes the mode to start the WebSocket connector. Can be "client" or "server"

Reply:
done